docs/users/installation.md

```diff
@@ -87,17 +113,71 @@ git diff // should produce a diff
 
 ### Settings and tasks
 
-| Name               | Type           | Description                                                                                                                      |
-| ------------------ | -------------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| `scalafix <args>`  | `Unit`         | Invoke scalafix command line interface directly. See [--help](#help) use tab completion to explore supported arguments           |
-| `scalafixConfig`   | `Option[File]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory if it exists. |
+| Name              | Type                       | Description                                                                                                                       |
+| ----------------- | -------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `scalafix <args>` | `TaskKey[Unit]`            | Invoke scalafix command line interface directly. Use tab completion to explore supported arguments or consult [--help](#help)     |
+| `scalafixConfig`  | `SettingKey[Option[File]]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory, if it exists. |
+
+### Main and test sources
+
+The task `myproject/scalafix` runs only in the project `myproject` for main
+sources, excluding test sources. To run Scalafix on test source, execute
+`myproject/test:scalafix` instead. To run on both main and test sources, execute
+`all myproject/scalafix myproject/test:scalafix`
 
-### FAQ
+### Integration tests
 
-- How to exclude/add files that are processed by the `scalafix` task?
+By default, the `scalafix` command is only enabled for the `Compile` and `Test`
+configurations. To enable Scalafix for other configuration like
+`IntegrationTest`, add the following to your project settings
 
+```scala
+lazy val myproject = project
+  .configs(IntegrationTest)
+  .settings(
+    Defaults.itSettings,
+    inConfig(IntegrationTest)(scalafixConfigSettings(IntegrationTest)),
+  // ...
+)
+```
+
+### Multi-module builds
+
+The `scalafix` task aggregates like the `compile` and `test` tasks. To run
+Scalafix on all projects for both main and test sources you can execute
+`all scalafix test:scalafix`.
+
+Optionally, add a command alias to your build to run Scalafix on your entire
+project with the shorthand `fix`
+
+```scala
+// top of build.sbt
+addCommandAlias("fix", "all compile:scalafix test:scalafix")
+```
+
+### Enforce in CI
+
+To automatically enforce that Scalafix has been run on all sources, use the
+`scalafixTest` instead of `scalafix`. This task fails the build if running
```

Comment:
This task does not exist yet, I will add it later https://github.com/scalacenter/scalafix/issues/846

Reply:
@olafurpg can you explain me again why we need `scalafixTest` and we cannot use `scalafix --test`?

Reply:
`scalafixTest` is needed in order to support the parallel `all scalafixTest test:scalafixTest`. Otherwise, you would need to run the sequential `; scalafix --test; test:scalafix --test`.

Reply:
I've removed `scalafixTest` in favor of `scalafix --test`. I'm convinced having a single `scalafix` task is the best option.